### ISSUE TITLE
Update default for the Version property

### ISSUE THREAD

**Issue by jhorsman** [pkjaer/tridion-powershell-modules#51]
The CoreService module [defaults to Version 2011-SP1](https://github.com/pkjaer/tridion-powershell-modules/blob/develop/CoreService/Settings.psm1#L42) (SDL Tridion 2011 SP1). 
Tridion Sites 9.1 no longer supports this endpoint and when using the PowerShell CoreService module with the default configuration you will get an error message.

I think it's time to update the default Version. For example to "Web-8.5" which is the latest supported Sites version (under limited support).

**Comment by pkjaer** [comment]
I agree. The idea was that it would use the lowest possible version so that it would work immediately (especially because once it's loaded a client it cannot unload it again). That's obviously broken in 9.1 so maybe I'll change the default to 2013? There's still a lot of people on those old versions, unfortunately.

Ideally, it would be able to first do a call to get the API version and then base the client on that (but again, clients cannot be unloaded so it would have to do it without a client). I might attempt something like that.

From 9.5 onward I want to use the REST API instead so it can be run in PowerShell Core. So the current approach of supporting all versions in one might not be very future proof. Will give it some more thought..

**Comment by jhorsman** [comment]
Changing it to 2013 sounds good as well. That will work for most implementations.

I have looked at this a while back. The DLLs are loaded in a creative way, I think to be able to keep multiple versions of the same DLL. Otherwise indeed the PowerShell module could attempt to connect and fallback to an older version. It would be great if we could un-load a CoreService DLL.

With the many authentication options we have, it sounds like a lot of work to make a custom mechanism to ping the Content Manager server and figure out what version it is.
